src/PyCall.jl

```diff
@@ -17,7 +17,7 @@ import Base: size, ndims, similar, copy, getindex, setindex!, stride,
        convert, pointer, summary, convert, show, haskey, keys, values,
        eltype, get, delete!, empty!, length, isempty,
        filter!, hash, splice!, pop!, ==, isequal, push!,
-       append!, insert!, prepend!, unsafe_convert
+       append!, insert!, prepend!, unsafe_convert, getproperty, propertynames
```

Comment:
Rather than importing, I'd prefer to move towards the style of overriding `Base.getproperty` etcetera, which is nowadays usually the preferred style in Julia.

Reply:
Done.